jest.config.js

```diff
@@ -30,4 +33,5 @@ module.exports = {
   testEnvironmentOptions: {
     url: 'http://localhost/',
   },
-};
+  testTimeout: 25000,
```

Comment:
Is this timeout strictly necessary? Things like this tend to make builds take a long time (since it potentially pauses for 25 seconds for a unit tests, which feels excessive).